package.json

```diff
@@ -19,6 +19,7 @@
     "copy-webpack-plugin": "^13.0.1",
     "css-loader": "^7.1.2",
     "css-minimizer-webpack-plugin": "^7.0.2",
+    "embla-carousel": "^8.6.0",
```

Comment:
(blocking) This would be the first time we publish 3rd party npm content to users on our site. @stevejalim do we have rules around that.

Reply:
Update - we discussed this in Slack and it's good to include 👍